Kentico.Kontent.Management/Models/Environments/EnvironmentClonedModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace Kentico.Kontent.Management.Models.Environments
+{
+    /// <summary>
+    /// Represents result of environment cloning.
+    /// </summary>
+    public class EnvironmentClonedModel
+    {
+        /// <summary>
+        /// Gets or sets the ID of the cloned environment.
+        /// </summary>
+        [JsonProperty("id")]
+        public Guid Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Management API key.
+        /// </summary>
+        [JsonProperty("management_api_key")]
+        public string ManagementApiKey { get; set; }
```

Comment:
Is this safe? As we only support HTTPS for MAPI I guess so...

Reply:
We consulted this with the security team when we designed the endpoint and they consider that safe.